src/hooks/Guilds/useIsProposalCreationAllowed/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTypedParams } from 'Modules/Guilds/Hooks/useTypedParams';

```

Comment:
If you pull the latest `develop` you can then move this file into `Modules/Guilds/Hooks/` instead since we are starting phase out guild hooks from this folder.